src/Apps/Sell/Routes/SubmissionRoute.tsx

```diff
@@ -28,7 +29,10 @@ export const SubmissionRoute: React.FC<SubmissionRouteProps> = props => {
       <AppContainer>
         <SellMeta />
 
-        <SellFlowContextProvider submissionID={submission.externalId}>
+        <SellFlowContextProvider
```

Comment:
**suggestion**: The SellFlowContextProvider could define it's own submission data needs in a fragment which is included in the query. Could then use one prop:

```tsx
<SellFlowContextProvider submission={submission}>
```

Reply:
That makes sense! Will also be helpful https://github.com/artsy/force/pull/14188/files#r1672486075